alchemy/test/cloudflare/worker.test.ts

```diff
@@ -1356,4 +1357,433 @@ describe("Worker Resource", () => {
+      const workerName = `${BRANCH_PREFIX}-test-worker-kv-${isWFP ? 'wfp' : 'std'}`;
+      const kvNamespace = new KVNamespace(`${workerName}-kv`);
+
+      let worker: Worker | undefined;
+      try {
+        // Create a worker with KV bindings
+        worker = await Worker(workerName, {
+          name: workerName,
+          script: `
+            export default {
+              async fetch(request, env, ctx) {
+                const url = new URL(request.url);
+                
+                if (url.pathname === '/set') {
+                  const key = url.searchParams.get('key') || 'test';
+                  const value = url.searchParams.get('value') || 'value';
+                  await env.MY_KV.put(key, value);
+                  return new Response(JSON.stringify({ action: 'set', key, value }), {
+                    headers: { 'Content-Type': 'application/json' }
+                  });
+                }
+                
+                if (url.pathname === '/get') {
+                  const key = url.searchParams.get('key') || 'test';
+                  const value = await env.MY_KV.get(key);
+                  return new Response(JSON.stringify({ 
+                    action: 'get', 
+                    key, 
+                    value,
+                    platform: ${isWFP ? 'true' : 'false'}
+                  }), {
+                    headers: { 'Content-Type': 'application/json' }
+                  });
+                }
+                
+                return new Response('KV Worker is running!', { status: 200 });
+              }
+            };
+          `,
+          format: "esm",
+          platform: isWFP,
+          bindings: {
+            MY_KV: kvNamespace,
+          },
+        });
+
+        // Verify worker properties
+        expect(worker.id).toBeTruthy();
+        expect(worker.name).toEqual(workerName);
+        expect(worker.platform).toEqual(isWFP);
+        expect(worker.bindings?.MY_KV).toBeDefined();
+
+        // Test KV functionality with HTTP requests
+        if (worker.url) {
+          // Set a value
+          const setResponse = await fetchAndExpectOK(`${worker.url}/set?key=testkey&value=testvalue`);
+          const setData = await setResponse.json();
+          expect(setData).toEqual({ action: 'set', key: 'testkey', value: 'testvalue' });
+
+          // Get the value
+          const getResponse = await fetchAndExpectOK(`${worker.url}/get?key=testkey`);
+          const getData = await getResponse.json();
+          expect(getData).toEqual({
+            action: 'get',
+            key: 'testkey',
+            value: 'testvalue',
+            platform: isWFP
+          });
+        }
+
+      } finally {
+        await destroy(scope);
+        await assertWorkerDoesNotExist(workerName, isWFP);
+      }
+    }
+  );
+
+  // Register the binding tests
+  for (const bindingTest of bindingTests) {
+    test(bindingTest.name, bindingTest.handler);
+  }
```

Comment:
You misunderstood. I want to remove these new tests you added and modify the worker.test.te, durable-object.test.ts etc tests that already exist to use a WFP variant. 

Go through all existing worker tests and change it to use the testBothPlatforms helper and modify its test logic to be parameterized by a label "script" or "wfp"